lib/broc-component-css-preprocessor.js

```diff
@@ -98,7 +100,20 @@ BrocComponentCssPreprocessor.prototype.write = function (readTree, destDir) {
         var podGuid = podPath.join('--') + '-' + guid();
         var fileContents = fs.readFileSync(path.join(srcDir, filepath)).toString();
 
-        buffer.push(processors[pod.extension](fileContents, podGuid));
+        if(pod.extension !== 'css') {
+          var outFilepath = filepath;
+          if (useComponentNameAsFileName) {
+            outFilepath = path.join('pod-styles', podPath.join('/') + '.' + pod.extension);
+          }
+
+          mkdirp.sync(path.join(destDir,path.dirname(outFilepath)));
+          fs.writeFileSync(path.join(destDir, outFilepath), processors[pod.extension](fileContents, podGuid));
+
+          buffer.push('@import "' + outFilepath + '"; \r\n');
```

Comment:
@import works for stylus, sass, and less at least. I think we could add some logic here if its different for future preprocessors. But it works for the ones we support currently.